docs/ja/docs/fastapi-people.md

```diff
@@ -0,0 +1,136 @@
+{% endfor %}
+
+</div>
+
+{% endif %}
+
+私は **FastAPI** の作成者および Maintainer です。詳しくは [FastAPIを応援 - ヘルプの入手 - 開発者とつながる](help-fastapi.md#開発者とつながる){.internal-link target=_blank} に記載しています。
+
+...ところで、ここではコミュニティを紹介したいと思います。
+
+---
+
+**FastAPI** は、コミュニティから多くのサポートを受けています。そこで、彼らの貢献にスポットライトを当てたいと思います。
+
+紹介するのは次のような人々です:
+
+* [GitHub issuesで他の人を助ける](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}。
+* [プルリクエストをする](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}。
+* プルリクエストのレビューをする ([特に翻訳に重要](contributing.md#translations){.internal-link target=_blank})。
+
+彼らに大きな拍手を。👏 🙇
+
+## 先月最もアクティブだったユーザー
+
+彼らは、先月の[GitHub issuesで最も多くの人を助けた](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}ユーザーです。☕
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.last_month_active %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Exparts
+
+**FastAPI exparts** を紹介します。🤓
+
+彼らは、*これまでに* [GitHub issuesで最も多くの人を助けた](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}ユーザーです。
+
+多くの人を助けることでexpartsであると示されています。✨
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.experts %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Top Contributors
+
+**Top Contributors** を紹介します。👷
+
+彼らは、*マージされた* [最も多くのプルリクエストを作成した](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}ユーザーです。
+
+ソースコード、ドキュメント、翻訳などに貢献してくれました。📦
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.top_contributors %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+他にもたくさん (100人以上) の contributors がいます。<a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">FastAPI GitHub Contributors ページ</a>ですべての contributors を確認できます。👷
+
+## Top Reviewers
+
+以下のユーザーは **Top Reviewers** です。🕵️
+
+### 翻訳のレビュー
+
+私は少しの言語しか話せません (もしくはあまり上手ではありません😅)。したがって、reviewers は、ドキュメントの[**翻訳を承認する権限**](contributing.md#translations){.internal-link target=_blank}を持っています。それらがなければ、いくつかの言語のドキュメントはありません。
```

Comment:
"いくつかの言語のドキュメントはなかったでしょう。" is a little better because the sentences "Without them, there wouldn't be..." is probably subjunctive past.
※ "subjunctive past" = 仮定法過去
https://ja.wikipedia.org/wiki/接続法#仮定法過去